main.cpp

```diff
@@ -80,72 +116,120 @@ bool rentACar(int rentalDuration, int carType = 0)  //function declaration with
     return {}; //if your function returns something other than void, add 'return {};' at the end of it.
 } 
 
+    
+    
+   
+    
+    
+    
 /*
- 1)
+    1)i
  */
-
+    int setVolume(int db)
+    {
+         ignoreUnused(db); 
+         return {};
+    }
 /*
  2)
  */
-
+    void getAge(int firstDigit, int secondDigit)
+    {
+        ignoreUnused(firstDigit, secondDigit);
+        
+     }
 /*
  3)
  */
-
+    float setThreshold(int upper, int lower)
+    { 
+        ignoreUnused(upper, lower );
+        return {}; 
+    }
 /*
  4)
  */
-
+    double setFiltFreq(double freqHz)
+    {
+         ignoreUnused(freqHz);
+         return {};  
+    }
 /*
  5)
  */
-
+    bool isFilt2Active(bool onFilt)
+    {
+         ignoreUnused(onFilt); 
+         return {}; 
+    }
 /*
  6)
  */
-
+    int getLfoRate(int a = 80, int b = 90)
+    {
+         ignoreUnused(a, b); 
+         return {}; 
+    }
 /*
  7)
  */
-
+    char setFiltType(char BP, char LP, char HP)
+    {
+         ignoreUnused(BP, LP, HP); 
+         return {}; 
+    }
+    
 /*
  8)
  */
-
+    double getLfoDepth(double lfo1, double lfo2 )
+    {
+         ignoreUnused(lfo1, lfo2); 
+         return {}; 
+    }
+    
 /*
  9)
  */
-
+    void val1(bool a, bool b) 
+    {
+         ignoreUnused(a, b); 
+          
+    }
 /*
  10)
  */
-
+    char signal1(int signA = 1, bool clear = false)
+    {
+         ignoreUnused(signA, clear); 
+         return {}; 
+    }
 int main()
 {
     //example of calling that function, storing the value, and passing it to ignoreUnused at the end of main()
     auto carRented = rentACar(6, 2); 
     
     //1)
-    
+    setVolume(8);
     //2)
-    
+    getAge(4, 9);
     //3)
-    
+    auto getCompression = setThreshold(2, 9);
     //4)
-    
+    setFiltFreq(333.33);
     //5)
-    
+    isFilt2Active(true);
     //6)
-    
+    getLfoRate();
     //7)
-    
+    setFiltType('b', 'l', 'h');
     //8)
-    
+    getLfoDepth(3.5, 6.2);
     //9)
-    
+    val1(true, true) ;
     //10)
-    
-    
+    signal1(2,true);
+    ignoreUnused(setVolume, getAge, getCompression, setThreshold, setFiltFreq, isFilt2Active, getLfoRate, setFiltType, getLfoDepth, val1, signal1);
```

Comment:
If any of these functions return a value, you are supposed to store the value it returns in a variable, and pass _that_ variable to `ignoreUnused`.  you're not supposed to pass the function to `ignoreUnused`, which is what you have done here.
- [ ] revise.